cmds/grpc-backend/main.go

```diff
@@ -102,7 +103,7 @@ func createKeyResolver() (auth.KeyResolver, error) {
 	}
 }
 
-func createRIDServer(ctx context.Context, locality string, logger *zap.Logger) (*rid.Server, error) {
+func createRIDServer(ctx context.Context, locality string, logger *zap.Logger, garbageColletorSpec string) (*rid.Server, error) {
```

Comment:
Passing it around is nice its a global variable and you are accessing it in this module you can just access it directly with `*garbageCollectorSpec`